CourseDevelopment/Week5/Chapter5.MD

```diff
@@ -1,104 +1,181 @@
 # Course Development Week 5
-Course Development Week 5 for the Open Science Course for physics students at graduate level. This document is based on the Syllabus.MD.
+Lecture plan for Week 5 of the OS4P: Open Science Course for physics students at graduate level.
 
-Date: Dec/07/2022
+last updated: 05/06/2023
 
-Name: Hendrik Snijder @hendelhendel
+Responsible author: Sanli @sanlifaez
 
-Open tasks @SanliFaez @erikvansebille
-+ Feedback on lecture-tutorial distribution of topics
-+ Feedback on lecture format
-+ Feedback on tutorial format
+contributors: 
+- Hendrik Snijder @hendelhendel
+- Erik van Sebille @erikvansebille
 
-## Goal: public engagement: story-telling and presentation; activism and trust in scientists
-After this week, students understand the role, responsibilities and interaction of scientists with scientific communities and the general public. 
-Students have to give a pitch about their other assignments using skills learned in the course. 
-Further, the students are able to formulate their opinion on the role a scientist must take in society, supported by arguments learned in class. By taking into account different stakes in the public debate, they can individually reflect critically on other viewpoints.
+Open tasks @SanliFaez 
++ Pick one lecture among the available option
++ Write down the reproducibility exercise
++ Specify the requirements for the open hardware measurement project
 
-## Keywords student familiarity check week 5
-Some of the keywords in [the syllabus](https://github.com/SanliFaez/OS4Physicists/blob/main/Syllabus_202x.md#week-5-public-engagement-story-telling-and-presentation-activism-and-trust-in-scientists) are probably familiar for students, while some others are new.
+## Goal: open design and open hardware; prototyping? 
+After this week, one is familier witht he concept of openscience in hardware development. Students get familiar with finding open hardware recipies and know how to contribute to them. After the lecture students can explain in a discussion what challenges open hardware can solve and what socio-economic issues are involved. 
+
+
+![Frontpage](LINK TO PICTURE)
+![image](https://user-images.githubusercontent.com/93695286/221529025-cc8a838e-f1d7-4a2d-b538-8bc0a939a538.png)
+
+
+
+## Keywords
 
 |**New Concepts**|**Familiar Concepts**|
```

Comment:
Again, if all concepts are 'new', then why present this in a table?

Reply:
this is suggested as an example for the exercise each student must do. I can put a hint that is related to the in-class exercise, or move it to the lecture slides

Reply:
I see that the other lectures show this table differently. I think we should review it once all the lectures are complete.